dask/dataframe/core.py

```diff
@@ -2671,7 +2731,7 @@ def sem(self, axis=None, skipna=True, ddof=1, split_every=False, numeric_only=No
                 result.divisions = (self.columns.min(), self.columns.max())
             return result
 
-    def quantile(self, q=0.5, axis=0, method="default"):
+    def quantile(self, q=0.5, axis=0, numeric_only=True, method="default"):
```

Comment:
Looking at pandas  to check for consistency. this default will be a no_default in 1.5, and it says in the future it'll be False. see https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.quantile.html

Do we need to warn here too?